include/powsybl/stdcxx/reference_wrapper.hpp

```diff
@@ -29,7 +29,7 @@ class reference_wrapper {
 
     reference_wrapper(const reference_wrapper&) = default;
 
-    reference_wrapper(T&& reference) noexcept = delete;
+    reference_wrapper(T&&) noexcept = delete;
```

Comment:
Maybe, to avoid conflicts, you should revert this change and do it in the PR relative to reference_wrapper

Reply:
Agree, it is fixed in PR #266.